src/metric.ts

```diff
@@ -0,0 +1,27 @@
+export class Metric {
```

Comment:
I don't think this class name needs to be so general. We can makes this name clearer and more specific with something like "FrameCounter".